scripts/linters/js_ts_linter.py

```diff
@@ -141,11 +141,11 @@ def compile_all_ts_files() -> None:
     run. For more details, please see issue #9458.
     """
     cmd = ('./node_modules/typescript/bin/tsc -p %s -outDir %s') % (
-        './tsconfig-lint.json', COMPILED_TYPESCRIPT_TMP_PATH)
-    proc = subprocess.Popen(
-        cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
-
-    _, encoded_stderr = proc.communicate()
+        './tsconfig.json', COMPILED_TYPESCRIPT_TMP_PATH)
```

Comment:
Unexpected change on this line.

Reply:
I have changed that line back to what it was before now.